src/main/java/org/jenkinsci/plugins/workflow/cps/CpsBodyExecution.java

```diff
@@ -86,15 +84,9 @@
     @GuardedBy("this")
     private Outcome outcome;
 
-    /**
-     * @see CpsBodyInvoker#createBodyBlockNode
-     */
-    private final boolean createBodyBlockNode;
-
-    public CpsBodyExecution(CpsStepContext context, List<BodyExecutionCallback> callbacks, boolean createBodyBlockNode) {
+    CpsBodyExecution(CpsStepContext context, List<BodyExecutionCallback> callbacks) {
```

Comment:
No. The class as a whole was not `public`, so no one could have been calling this constructor from outside anyway. (Not that we would want to support them if they did.)